x-pack/plugins/watcher/__jest__/client_integration/watch_create_threshold.test.tsx

```diff
@@ -23,6 +21,15 @@ const MATCH_INDICES = ['index1'];
 
 const ES_FIELDS = [{ name: '@timestamp', type: 'date' }];
 
+// Since watchID's are dynamically created, we have to mock
+// the function that generates them in order to be able to match
+// against it.
+jest.mock('uuid/v4', () => {
+  return function () {
+    return '12345';
```

Comment:
I noticed some of the other tests use the `WATCH_ID` constant - does it make sense to use that here (and anywhere else it is hard-coded)?